api/application_round_api.py

```diff
@@ -1,10 +1,14 @@
 import logging
 
 from django.conf import settings
-from django.db.models import Sum
+from django.db.models import Case, Exists, OuterRef, Prefetch, Subquery, Sum
+from django.db.models import Value as V
```

Comment:
Why `V`? I think `Value` is good as is.

Reply:
This is what the Django documentation recommends. Have not putten much thought to why, but I would guess that it clashes with some very common library or function. For example in the `Concat` database function: https://docs.djangoproject.com/en/4.0/ref/models/database-functions/#concat